en/Home.md

```diff
@@ -10,6 +10,7 @@ Welcome to the official Obsidian documentation, where you can find tips and guid
 
 You can browse the documentation in any of the following languages:
 
+- [العربية](https://publish.obsidian.md/help-ar)
```

Comment:
Could you add it to the end please? We may do a re-org at some point, but currently they're sorted by date completed. Thanks!!